### DISCUSSION TITLE
Ubuntu 20.04 Support

### DISCUSSION THREAD

**Discussion by ArmyGeekHack** [telekom-security/tpotce#777] [Q&A]
Hello! I am trying to install T-pot on Ubuntu Focal Fossa and its not supported. Is there a way to do this or will it be supported soon? Thank you!!

**Comment by t3chn0m4g3** [comment]
After we have left Ubuntu for Debian about 2 years ago there are no immediate plans to support it again.

**Reply by ppatrik** [reply]
Hello, if i can ask what was the reason to drop Ubuntu support? Are there features needed for running tpot like firewall configuration or network configuration that are not available in Ubuntu distribution? We are planning ansible instalation of few tpots and until now we have better skills and better upgradeability with this distribution.

Thanks in advance
Patrik

**Comment by t3chn0m4g3** [comment]
In a direct comparison Debian ran signifacantly more stable than Ubuntu (server) WRT the T-Pot setup.
